deltametrics/mask.py

```diff
@@ -106,28 +368,199 @@ def a_channel_function(self):
         return a_channel_function(self.data)
 
 
-class LandMask(BaseMask):
-    """land-water mask.
+class WetMask(BaseMask, OAM):
+    """Compute the wet mask.
+
+    A wet mask object, identifies all wet pixels on the delta topset. Starts
+    with the land mask and then uses the topo_threshold defined for the
+    shoreline computation to add the wet pixels on the topset back to the mask.
 
-    A land-water mask object, helps enforce valid masking of land water interface.
+    If a land mask has already been computed, then it can be used to define the
+    wet mask. Otherwise the wet mask can be computed from scratch.
 
     Examples
     --------
+    Initialize the wet mask
+        >>> wmsk = dm.mask.WetMask(arr)
```

Comment:
same as above, what exactly is `arr`? make it explicit by pulling from the `rcm8cube`

Reply:
Took care of this and all other mask code examples.